src/PrestaShopBundle/Form/Admin/Sell/Product/Stock/AvailabilityType.php

```diff
@@ -75,6 +78,19 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'type' => TextType::class,
                 'label' => $this->trans('Label when in stock', 'Admin.Catalog.Feature'),
                 'required' => false,
+                'options' => [
+                    'constraints' => [
+                        new TypedRegex(TypedRegex::TYPE_GENERIC_NAME),
+                        new Length([
```

Comment:
I am not sure if there is a guideline here, but I've been working with something similar on https://github.com/PrestaShop/PrestaShop/pull/29396 and came across https://github.com/PrestaShop/PrestaShop/blob/c9676163c1841900ab6e1c79628b114863ce93c6/src/PrestaShopBundle/Form/Admin/Product/ProductInformation.php#L228. Where it uses symfony default message that can handle pluralization

Reply:
hmm so we don't need to provide the custom message anymore? (I haven't checked it if it would work, just copied from other examples)

Reply:
I guess it's a question we should ask @Julievrz or @l-delin 
Do you think we should rely on Symfony inner wordings (which are automatically translated) or should we always use our custom wordings? So they need to be translated in Crowdin and/or in the BO interface.

Reply:
As @yanmakouf said, I don't think we need to prioritise either - we can keep going as usual. If the wording is fine we keep it, if it's not we'll edit it.

Reply:
IMO all wordings should go through our own translation component because Symfony's internal translations cannot be customized through our translation interface.